frontend/src/GlobalStyle.tsx

```diff
@@ -117,4 +118,6 @@ const GLOBAL_STYLE = css({
         border: "none",
         borderTop: `1px solid ${COLORS.grey4}`,
     },
+    "p, label": { color: COLORS.grey7 },
```

Comment:
The text problem in the error box is due to this, as it overrides the `color` of the error box. Why did you add this general rule here anyway?

Reply:
Because I'm lazy of course. I believe I wanted to darken some labels on the "page settings" and "add sub page" pages because they seemed a little bright. 
But without this they use `COLORS.foreground` which is alright, so this really isn't needed. I think/hope that the `p` and `input` rule can stay there, but I could also apply that on the individual elements (and there it would also be possible to make the color conditional on the color scheme, so that might be an upside).

Though `color: COLORS.foreground` (which is applied to the body further up in this file) still doesn't work for the error message, so that also needs to be fixed individually.